pkg/services/provisioning/provisioning_test.go

```diff
@@ -92,7 +93,7 @@ func setup() *serviceTestStruct {
 	}
 
 	serviceTest.service = newProvisioningServiceImpl(
-		func(context.Context, string, dboards.Store) (dashboards.DashboardProvisioner, error) {
+		func(context.Context, string, dboards.Store, utils.OrgStore) (dashboards.DashboardProvisioner, error) {
```

Comment:
I am thinking should we have a store per service or we should do store composition like this, do you have a preference?